src/app/_components/CheckBox/CheckBox.js

```diff
@@ -0,0 +1,172 @@
+'use client';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from '@/utils/cn';
+import { checkBoxVariants } from './styles';
+
+export default function CheckBox({
+  value,
+  onChange,
+  name,
+  text = '',
+  subText = '',
+  disabled = false,
+  checked = false,
+  children = null,
+  variant = checkBoxVariants.default,
+}) {
+  const id = `checkbox_id_${name}_|_${value}__`;
```

Comment:
@IvanKoshan Consider simplifying the `id` format if the extra characters were not added for a specific purpose.

Reply:
Is it alright now @yataras ?